applications/nrf5340_audio/src/bluetooth/ble_audio_services.h

```diff
@@ -88,16 +76,78 @@ int ble_vcs_volume_mute(void);
  *         the target device will be itself.
  *
  * @return 0 for success,
- *         -ENXIO if the feature is disabled
+ *         -ENXIO if the feature is disabled.
  *         Other errors from underlying drivers.
  */
 int ble_vcs_volume_unmute(void);
 
 /**
- * @brief Initialize the Volume Control Service server
+ * @brief  Discover VCS and included services.
+ *
+ * @param  conn Pointer for peer connection information.
+ * @param  channel_num The number of the remote device.
+ *
+ * This will start a GATT discovery and setup handles and subscriptions for
+ * VCS and included services.
+ * This shall be called once before any other actions related with VCS.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int ble_vcs_discover(struct bt_conn *conn, uint8_t channel_num);
+
+/**
+ * @brief  Discover MCS and included services.
+ *
+ * @param  conn  Pointer to the active connection, only valid for client.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int ble_mcs_discover(struct bt_conn *conn);
+
+/**
+ * @brief  Get the current state of the media player, only valid for client.
+ *
+ * @param  conn  Pointer to the active connection.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int ble_mcs_state_update(struct bt_conn *conn);
+
+/**
+ * @brief  Send play/pause command to the media player,
+ *         depending on the current state.
+ *
+ * @param  conn  Pointer to the active connection, only valid for client.
+ *               Shall be NULl if called from server.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int ble_mcs_play_pause(struct bt_conn *conn);
+
+/**
+ * @brief Initialize the Volume Control Service client.
+ */
+int ble_vcs_client_init(void);
```

Comment:
The order of functions in this header (and the corresponding source) should be changed. Here we have mcs, vcs, vcs, then mcs and mcs again.

Reply:
Init functions at the bottom. So we have general vcs functions, general mcs functions, init vcs functions, and lastly init mcs functions